debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/SqlUtils.java

```diff
@@ -335,21 +412,4 @@ public static long parseRetentionFromName(String historyTableName) {
                 Integer.parseInt(tokens[6])); // minutes
         return Duration.between(recorded, LocalDateTime.now()).toHours();
     }
-
-    /**
-     * This method builds table_name IN predicate, filtering out non whitelisted tables from Log Mining.
-     * It limits joining condition over 1000 tables, Oracle will throw exception in such predicate.
-     * @param tables white listed table names
-     * @return IN predicate or empty string if number of whitelisted tables exceeds 1000
```

Comment:
Do we still have this consideration of the max IN size somewhere?

Reply:
No.  We use the Oracle `REGEXP_LIKE` operator that takes each of the include/exclude list regex patterns and applies a single predicate to the where-clause for each pattern.

Reply:
Wow, I see. I'm a tad concerned there may be subtle differences in the regexp engine semantics between Java and Oracle's, potentially resulting in inconsistencies between connectors. So is filter handling for Oracle then _exclusively_ done via that database function?

Reply:
For LogMiner during streaming, yes filtering is handled exclusively by the database query itself.  For all other portions of the code, including Xstreams or during Snapshotting, this is handled in Java exclusively.